src/commands/copy/__test__/copy.test.ts

```diff
@@ -151,4 +151,140 @@ describe('copyFiles', () => {
     assert.equal(tiffSource?.contentType, 'binary/octet-stream');
     assert.equal(tiffTarget?.contentType, 'binary/octet-stream');
   });
+
+  it('should not copy files when different size or multihash but no force', async () => {
+    await Promise.all([
+      fsa.write('memory://source/topographic.json', Buffer.from(JSON.stringify({ test: true })), {
+        contentType: 'application/octet-stream',
+        metadata: {
+          multihash: '12206fd977db9b2afe87a9ceee48432881299a6aaf83d935fbbe83007660287f9c2e',
+          unique: 'fileA',
+        },
+      }),
+      fsa.write('memory://target/topographic.json', Buffer.from(JSON.stringify({ test: true })), {
```

Comment:
Since this file has a different multihash, it should also have different content for clarity. You could pull out for example `const originalContent = Buffer.from(JSON.stringify({ "a": 1 }))`, `const originalSizeDifferentContent = Buffer.from(JSON.stringify({ "b": 2 }))`, and `const differentSizeContent = Buffer.from(JSON.stringify({ "xxx": 123 }))` and their hashes to use in the various tests.